app/js/lib/server/network.js

```diff
@@ -84,14 +84,18 @@ export default class Network extends EventDispatcher {
       .then((response) => response.blob());
   }
 
+  fetch(url, method, body) {
+    return this[p.fetch](url, 'application/json', method, body);
+  }
+
```

Comment:
This method doesn't try to parse the response. Tell me id you'd prefer that fetchJSON use it as well.